File: 2315-count-asterisks/2315-count-asterisks.py
class Solution(object):
    def countAsterisks(self, s):
        """
        :type s: str
        :rtype: int
        """
        splits = s.split("|")
        print(splits)
        onlyEven = splits[::2]
            
        newstring = ''.join(onlyEven)
        return newstring.count('*')
        

File: 2315-count-asterisks/2315-count-asterisks.java
class Solution {
    public int countAsterisks(String s) {
        int symbolCount = 0, answer = 0;
        for(int i=0;i<s.length();i++){
            if(s.charAt(i) == '|'){
                symbolCount ++;
            }
            if(symbolCount%2==0 && s.charAt(i)=='*'){
                answer++;
            }
        }
        return answer;
    }
}